source/time.tex

```diff
@@ -9476,9 +9455,8 @@
 
 \pnum
 \effects
-Constructs a \tcode{zoned_time} by
-initializing \tcode{zone_} with \tcode{traits::default_zone()} and
-default constructing \tcode{tp_}.
+Initializes \tcode{zone_} with \tcode{traits::default_zone()} and
+default constructs \tcode{tp_}.
```

Comment:
default-initializes? (There are several occurrences of "default constructs" herein that I would prefer to be "default-initializes", I won't mark the remainder under the expectation that we'll consistently change all or none of them.)

Reply:
Make it a separate editorial issue, please. I'm trying hard to limit this pull request to the "blatantly obvious" changes, ignoring all the other wording nightmares in this section.